CSharp/Mazes/Grid/Grid.cs

```diff
@@ -2,6 +2,11 @@
 using System.Collections;
 using System.Collections.Generic;
 
+/*
```

Comment:
Use /// notation for comments.  

Remove the part about the 2D array.  People can see that I am using a 2D array from the code.

Move the description to `public sealed class Grid`  and not at the namespace level.